pyproject.toml

```
@@ -23,6 +23,7 @@ exclude = [ ]
   [tool.poetry.dependencies]
   python = "^3.9.0"
   pycryptodome = ">=3.18.0"
+  aiohttp = "^3.10.3"
```

```suggestion
  aiohttp = ">3.10.3"
```
Can we just make sure it is above the minimum version?